### ISSUE TITLE
Live Input Macros question-2

### ISSUE THREAD

**Issue by ektrontek** [chris-kent/Live-Input-Macros#10]
Dear Support Team, 
Because I found the information at the plugin introduce (refer to the link: https://marketplace.atlassian.com/plugins/net.kentcom.live-input-macros/server/support) , it describe as shown as the attachment.What does it mean ? Thanks you.
**These can be used to get feedback from users, or monitor tasks.
Use this for formas or getting feedback from users.**
![forum](https://user-images.githubusercontent.com/19470763/34359130-48256ec4-ea90-11e7-95fd-c7bba4f9d7e0.jpg)

Best Regards, Anderson Hsu

**Comment by chris-kent** [comment]
Hi there,

As this plugin was quite unique, we were not sure exactly how users would utilize the macros. These were possible suggestions as to the purpose/usefulness of the macros.  On reflection, using these in forms is not possible as they would need to store/collect and report on individual users selections and entries, which this does not do.  There are other plugins which do forms very well.

Basically, this plugin really just allows users to modify particular pieces of data without needing to Edit the page every time, assuming the data is from a preset selection, depending on the macro of course.

We hope this helps...

**Comment by ektrontek** [comment]
Dear Kent, 
Thanks for your kindly reply. Is there other plugin which about form application for a suggestion ? 
Best Regards, Anderson Hsu
